extension/src/background/helpers/account.ts

```diff
@@ -73,6 +77,10 @@ export const getSavedNetworks = () =>
   ) as NetworkDetails[];
 
 export const getNetworkDetails = () => {
+  if (!localStorage.getItem(NETWORK_ID)) {
+    localStorage.setItem(NETWORK_ID, JSON.stringify(DEFAULT_NETWORKS[0]));
+  }
```

Comment:
not super necessary, but I decided to add these in to make sure the "database" is matching the UI data